client/src/components/SelectionContainer.tsx

```diff
@@ -114,63 +113,65 @@ export default function SelectionContainer({
   };
 
   return (
-    <Grid container>
-      <Grid item xs={12} md={6} className={classes.grid}>
-        <Box className={classNames(classes.container)}>
-          <h2>Select Musicians</h2>
-          <div className={classNames(classes.musicians)}>
-            {musicians.map((musician: Musician) => (
-              <ItemCard
-                key={musician.musician_id}
-                item={musician}
-                onClick={() => handleClickItem(musician)}
-              />
-            ))}
-          </div>
-        </Box>
-      </Grid>
-      <Grid item xs={12} md={6}>
-        <Box className={classNames(classes.container)}>
-          <h2>Select Instruments</h2>
-          <div className={classNames(classes.musicians)}>
-            {instruments.map((instrument: Instrument) => (
-              <ItemCard
-                key={instrument.id}
-                item={instrument}
-                onClick={() => handleClickItem(instrument)}
-              />
-            ))}
-          </div>
-        </Box>
-        {/* only render link tag if selection criteria are met */}
-        {!isSelected || !selectedEqual ? (
-          <CustomButton
-            variant="contained"
-            disabled={!isSelected || !selectedEqual}
-          >
-            Assign
-          </CustomButton>
-        ) : (
-          <Link
-            href={{
-              pathname: "/assignments",
-              query: {
-                assignments: JSON.stringify(assign(musicians, instruments)),
-              },
-            }}
-          >
+    <>
```

Comment:
it's strange, I thought we had removed these in your previous PR. Let's match it to what's in the develop branch.

Reply:
Removed!